lms/static/scripts/frontend_apps/components/dashboard/SyncGradesButton.tsx

```diff
@@ -70,7 +88,17 @@ export default function SyncGradesButton({
       return (
         <>
           Syncing grades
-          <SpinnerCircleIcon className="ml-1.5" />
+          <div
+            className={classnames(
+              'border-solid border-l border-grey-5',
+              // Compensate the button's padding, so that this grows from top to
+              // bottom
+              '-my-2 self-stretch',
```

Comment:
Where is the padding defined and what does the component look like with/without the `-my-2`?

Reply:
I noticed a comment I created in another PR was lost here. That comment explains I I went with this approach: https://github.com/hypothesis/lms/pull/6751#discussion_r1787492866

> Where is the padding defined and what does the component look like with/without the `-my-2`?

The padding is defined in the Button component. Without this negative margin it looks like this:

![image](https://github.com/user-attachments/assets/4e63036c-1f23-4a29-b4a6-f21035346f51)

So this only affects the "separator", which we would usually render using different elements, but the comment I linked above explains I I went this way instead.

Reply:
Also, I have just updated the comment so that it says `the Button's padding`, with capital B, instead of `the button's padding`. That way it is more clear we are talking about our own `Button` component.

Reply:
> Compensate the button's padding, so that this grows from top to bottom

By "so that this grows from top to bottom", do you mean "so the separator covers the entire height of the button"?

Reply:
> > Compensate the button's padding, so that this grows from top to bottom
> 
> By "so that this grows from top to bottom", do you mean "so the separator covers the entire height of the button"?

Yes, exactly. Not the most clear wording 😅

Reply:
I changed the comment to `Compensate the Button's padding, so that this "separator" covers the entire height of the Button`